docsets/Vaadin/docset.json

```diff
@@ -12,6 +12,14 @@
         {
             "_comment": ""
         },
+        {
+            "version": "8.8.1",
+            "archive": "versions/8.7.2/Vaadin.tgz",
```

Comment:
This path seems to be wrong. It points to `8.7.2` instead of `8.8.1`.

Reply:
Whoops, yup... moving too fast! Will resubmit.